src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -86,10 +86,13 @@ function useChatUtilsHook(): UseChatUtilsHook {
 
   const goToCommunityScreen = (channel: ChannelList) => {
     setChannelAsRead(channel);
-
     const topicName = channel?.name?.replace(/^#/, '');
     const navigationParam = {
-      id: convertTopicNameToTopicPageScreenParam(topicName)
+      id: convertTopicNameToTopicPageScreenParam(topicName),
+      channelPicture: channel.channelPicture,
+      coverImage: channel.rawJson.cover_image,
+      isFollowing: channel.rawJson.is_following,
+      memberCount: channel.rawJson.topic_follower_count
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `navigationParam` object now includes more properties than before. Ensure that the 'TopicPageScreen' is capable of handling these additional properties and that they are used appropriately. If any of these properties can be null or undefined, make sure to handle those cases in the 'TopicPageScreen'.



<!-- This is an auto-generated comment by OSS CodeRabbit -->